modules/social_features/social_core/assets/css/admin-toolbar.scss

```diff
@@ -24,6 +24,18 @@ body[class*='gin'] {
 					}
 				}
 			}
+
+      .menu-item__social_core-dashboard {
+        a.toolbar-icon-social-core-dashboard {
+          &:before {
+            width: 24px;
+            height: 24px;
+            -webkit-mask-image: none;
+            mask-image: none;
+            background: url('../../images/icon-cog.svg') no-repeat 0/cover;
```

Comment:
![Screenshot 2022-02-10 at 12 37 45](https://user-images.githubusercontent.com/16667281/153401432-e1f7e9e4-d0aa-49fa-8172-c58b8ee3f5ff.png)

We're better of with the open cog item, also make sure that once you're hovering over this it takes in to account the active theme color (happy to show you)